napari/layers/utils/stack_utils.py

```diff
@@ -0,0 +1,103 @@
+import warnings
+from typing import Union, List
+
+import numpy as np
+from napari.layers import Image, Labels
+from vispy.color import Colormap
+
+
+def stack_to_images(
+    stack: Union[Image, Labels],
+    axis: int,
+    colormaps: List[Union[str, Colormap]] = None,
+) -> List[Image]:
+    """Function to split the active layer into separate layers along an axis
+
+    Parameters
+    ----------
+    stack : napari.layers.Image
+        The image stack to be split into a list of image layers
+    axis : int
+        The axis to split along.
+
+    Returns
+    -------
+    list
+        List of images
+    """
+
+    data = stack.data
+    name = stack.name
+    num_dim = len(data.shape)
+
+    if num_dim < 3:
+        warnings.warn(
+            "The image needs more than 2 dimensions for splitting", UserWarning
+        )
+        return None
+
+    if axis >= num_dim:
+        warnings.warn(
+            "Can't split along axis {}. The image has {} dimensions".format(
+                axis, num_dim
+            ),
+            UserWarning,
+        )
+        return None
+
+    cmap = stack.colormap
+    if colormaps is None:
+        colormaps = data.shape[axis] * [cmap]
+
+    imagelist = list()
+
+    for i in range(data.shape[axis]):
+        layer_name = "{:02d}_{}".format(i, name)
+
+        try:
+            color = colormaps[i]
+        except IndexError:
+            color = 'gray'
+
+        image = Image(
+            np.take(data, i, axis=axis),
+            blending='additive',
+            colormap=color,
+            name=layer_name,
+        )
+
+        imagelist.append(image)
+
+    return imagelist
+
+
+def images_to_stack(
+    images: List[Union[Image, Labels]], axis: int = -1
```

Comment:
It would be much more efficient to combine along `axis=0` by default... Not to mention, it would be guaranteed have the desired effect of adding a slider to the viewer, whereas the converse might result in rgb display (if there are three channels) or a super-thin slice being displayed from each channel (if there is any other number of channels).